services/api-server/src/simcore_service_api_server/api/dependencies/application.py

```diff
@@ -33,3 +20,12 @@ async def get_product_name(app: FastAPI = Depends(get_app)) -> str:
     default_product_name = app.state.default_product_name
     assert isinstance(default_product_name, str)  # nosec
     return default_product_name
+
+
+assert get_reverse_url_mapper  # nosec
+assert get_app  # nosec
+
+__all__: tuple[str, ...] = (
+    "get_reverse_url_mapper",
+    "get_app",
+)
```

Comment:
Q: why can't those dependencies be imported where they are used from servicelib?

Reply:
I use it as "cluster for common dependencies" in a service.  Note that all services have the same layout/skeleton.
let me follow up on that in coming PRs after having discussed a bit with all BE to find a consensus, ok?